types/dragino/t68dl/default.schema.json

```diff
@@ -0,0 +1,14 @@
+{
+  "$id": "https://akenza.io/dragino/t68dl/default.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "processingType": "uplink_decoder",
+  "topic": "default",
+  "title": "Default",
+  "properties": {
+    "temperature": {
+      "$ref": "https://raw.githubusercontent.com/akenza-io/device-type-library/main/data-models/environment/schema.json#/$defs/temperature/celsius"
+    }
```

Comment:
Please also describe the flags which are sent simultaneously with the temperature.